### PR TITLE
Add new assets and quickstart guide to Japanese documentation

### PR DESCRIPTION
**Pull request by funkyenough**
- Add new images (cargo-stylus.png and stylus-wallet.png) to assets
- Add translated stylus-quickstart.md file

### CODE REVIEW

File: website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/stylus-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+title: 'クイックスタート: Stylusを使用してRustでスマートコントラクトを開発'
+sidebar_label: 'クイックスタート (Rust, Stylus)'
+description: 'Leads a developer from 0 to 1 writing and deploying a smart contract in Rust using Stylus'
+author: chrisco512
+sme: chrisco512
+sidebar_position: 2
+target_audience: Developers writing Stylus contracts in Rust using Stylus
+---
+
+import PublicPreviewBannerPartial from './partials/_stylus-public-preview-banner.md';
```
[the remark this answers is not in the document]
Comment (funkyenough) [reply website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/stylus-quickstart.md]: Understood. 

- Added banner to `./partials/_stylus-public-preview-banner-partial.md`
- Changed import message to `import PublicPreviewBannerPartial from './partials/_stylus-public-preview-banner-partial.md';`

**Comment by hkalodner** [comment]
@cla-bot check

**Comment by hkalodner** [comment]
@cla-bot check

**Review by symbolpunk — APPROVED** [review]
Love the way this was integrated - thank you!

LGTM from a structural perspective, cc @Jason-W123 to merge whenever you're ready :)

File: website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/rust-sdk-guide.md [hunk 2]
```diff
@@ -0,0 +1,699 @@
...
+sidebar_label: 'Stylus Rust SDKオーバービュー'
+description: 'An in-depth overview of the features provided by the Stylus Rust SDK'
+author: rachel-bousfield
+translator: Ying@Fracton, Naoki@Fracton, Hikaru@Fracton
+sme: rachel-bousfield
+sidebar_position: 1
+target_audience: Developers using the Stylus Rust SDK to write and deploy smart contracts.
+---
+
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner.md';
```
Comment (Jason-W123) [inline website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/rust-sdk-guide.md:12]: There is no file named `_stylus-public-preview-banner.md`, you mean `_stylus-public-preview-banner.md_partial.md`?

File: website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/stylus-sdk.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+description: A table with links and minimal information about the different SDKs
+author: rachel-bousfield
+translator: Ying@Fracton, Naoki@Fracton, Yu@Fracton
+sme: rachel-bousfield
+target_audience: Developers writing and deploying Stylus programs
+sidebar_label: 'Stylus SDK レポジトリ'
+sidebar_position: 2
+---
+
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner.md';
```
Comment (Jason-W123) [inline website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/stylus-sdk.md:12]: Same as above

File: website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/testnet-information.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+description: A reference providing details about the Stylus testnet and faucets for obtaining testnet ETH
+author: amarrazza
+translator: Ying@Fracton, Naoki@Fracton, Hikaru@Fracton
+sme: amarrazza
+target_audience: Developers building on the Stylus testnet
+sidebar_label: 'テストネット情報'
+sidebar_position: 9
+---
+
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner.md';
```
Comment (Jason-W123) [inline website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/testnet-information.md:12]: Same here

File: website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/stylus-sdk.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+description: A table with links and minimal information about the different SDKs
+author: rachel-bousfield
+translator: Ying@Fracton, Naoki@Fracton, Yu@Fracton
+sme: rachel-bousfield
+target_audience: Developers writing and deploying Stylus programs
+sidebar_label: 'Stylus SDK レポジトリ'
+sidebar_position: 2
+---
+
+import TranslationBannerPartial from "./partials/_stylus-translation-banner-partial.md";
```
Comment (Jason-W123) [inline website/i18n/ja/docusaurus-plugin-content-docs/translated/stylus/reference/stylus-sdk.md:12]: If you want to import from `reference` dir, you should change the path to `../partials/xxx`. (Not only this file, but all files under `reference`)
```suggestion
import TranslationBannerPartial from "../partials/_stylus-translation-banner-partial.md";
```

**Review by Jason-W123 — CHANGES_REQUESTED** [review]

**Pull request merged**